src/ubx_message.toit

```diff
@@ -123,7 +126,7 @@ class Message:
 
   static is_valid_frame_ frame/ByteArray -> bool:
     // Check the sync bytes.
-    if frame[0] != 0xb5 or frame[1] != 0x62: throw INVALID_UBX_MESSAGE_
+    if frame[0] != 0xb5 or frame[1] != 0x62: return false
```

Comment:
This should probably behave like `Message.from_reader`, so `Message.from_byte` act the same.